src/features/goblins/bank/components/WithdrawItems.tsx

```diff
@@ -249,7 +249,16 @@ export const WithdrawItems: React.FC<Props> = ({
         </div>
 
         <span className="text-sm mb-4">
-          Once withdrawn, you will be able to view your items on OpenSea.
+          Once withdrawn, you will be able to view your items on OpenSea,{" "}
```

Comment:
```suggestion
          Once withdrawn, you will be able to view your items on OpenSea.{" "}
```

I think the capital letter on `Read more` looks a bit odd in the middle of the sentence 😅 

I'm not sure which one is the most suitable:

`Once withdrawn, you will be able to view your items on OpenSea, read more.`

`Once withdrawn, you will be able to view your items on OpenSea; Read more.`

`Once withdrawn, you will be able to view your items on OpenSea. Read more.`

Reply:
i think i will go for `Once withdrawn, you will be able to view your items on OpenSea. Read more.`